core/src/main/java/org/nervos/ckb/methods/response/CkbLockHash.java

```diff
@@ -0,0 +1,11 @@
+package org.nervos.ckb.methods.response;
+
+import org.nervos.ckb.methods.Response;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
+public class CkbLockHash extends Response<String> {
```

Comment:
```suggestion
public class CkbScriptHash extends Response<String> {
```

Is this script hash?

Reply:
OK. It is lock hash and also script hash.

Reply:
Script hash may be more general.